metallb/base/kustomization.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: metallb
+
+resources:
+  - https://raw.githubusercontent.com/metallb/metallb/v0.9.5/manifests/namespace.yaml
```

Comment:
Namespace has to go to `cluster-scoped` as well, not via a remote reference, but as a direct raw manifest.